operations/configuration_management/Troubleshoot_CFS_Sessions_Failing_to_Start.md

```diff
@@ -0,0 +1,81 @@
+# Troubleshoot CFS Sessions Failing to Start
+
+Troubleshoot issues where Configuration Framework Service \(CFS\) sessions are being created, but the pods are never created and never run.
+
+## Prerequisites
+
+CFS-batcher is creating automatic sessions, but pods aren't starting for those sessions.  There are a number of communication reasons this could be happening, so check the `cfs-batcher` and `cfs-operator` logs for these signatures.
```

Comment:
```suggestion
CFS-batcher is creating automatic sessions, but pods are not starting for those sessions. There are a number of communication reasons this could be happening, so check the `cfs-batcher` and `cfs-operator` logs for these signatures.
```